docs/en/sdk-manual/android.md

```diff
@@ -260,57 +260,57 @@ All user attributes will be stored in `user` object, and all custom and global a
-| app_title        | the app's display name                                                                        | generate from `android.content.pm.getApplicationLabel(appInfo)`                                                                                                                                                                                                  | data analysis needs                                                                                  |
+| attribute        | describe                                                                                      | how to generate                                                                                                                                                                                                                                                               | use and purpose                                                                                      |
+|------------------|-----------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------|
+| hashCode         | the event object's hash code                                                                  | generate from `Integer.toHexString(AnalyticsEvent.hashCode())`                                                                                                                                                                                                                | distinguish different event                                                                          |
+| app_id           | clickstream app id                                                                            | generated when clickstream app create from solution control plane.                                                                                                                                                                                                            | identify the events for your apps                                                                    |
+| unique_id        | the unique id for user                                                                        | generate from `UUID.randomUUID().toString()` when the sdk first initialization<br> it will be changed after user re-login to another user who never login, and when user re-login to the before user in same device, the unique_id will reset to the before user's unique_id. | the unique for identity different user and associating the behavior of logging in and not logging in |
```

Comment:
generated from xxx during the SDK initialization

Reply:
fixed.